plugins/aws/resoto_plugin_aws/resource/rds.py

```diff
@@ -463,6 +464,21 @@ def connect_in_graph(self, builder: GraphBuilder, source: Json) -> None:
                 builder.dependant_node(
                     self, reverse=True, delete_same_as_default=True, clazz=AwsEc2Subnet, id=subnet_id
                 )
+        if self.rds_kms_key_id:
+            if self.rds_kms_key_id.startswith("arn:"):
+                builder.dependant_node(from_node=self, clazz=AwsKmsKey, arn=self.rds_kms_key_id)
+            else:
+                builder.dependant_node(from_node=self, clazz=AwsKmsKey, id=self.rds_kms_key_id)
+        if self.rds_performance_insights_kms_key_id:
+            if self.rds_performance_insights_kms_key_id.startswith("arn:"):
+                builder.dependant_node(from_node=self, clazz=AwsKmsKey, arn=self.rds_performance_insights_kms_key_id)
+            else:
+                builder.dependant_node(from_node=self, clazz=AwsKmsKey, id=self.rds_performance_insights_kms_key_id)
```

Comment:
There is quite some repition.
Maybe we should add a static method in AwsKmsKey which encodes the handling if it is id or arn or something else?